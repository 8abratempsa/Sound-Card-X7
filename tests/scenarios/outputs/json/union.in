octosql "SELECT * FROM fixtures/union.json" --output json
