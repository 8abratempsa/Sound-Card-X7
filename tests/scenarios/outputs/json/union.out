{"union.id":0,"union.value":42}
{"union.id":0,"union.value":"42"}
{"union.id":0,"union.value":[42,"42"]}
